herd/tests/instructions/AArch64/A24.litmus

```diff
@@ -0,0 +1,11 @@
+AArch64 A24
+
+(* Tests ldrsb instruction *)
+{ 0:X0 = 1; 0:X1 = x; int8_t x = 127; }
+
+P0;
+  LDRSB X0, [X1];
+  MOV X2, #1;
+  ADD X0, X0, X2;
+
+exists (0:X0 = 129 /\ x = 0)
```

Comment:
A successful final condition with a `forall` quantifier would be appreciated. For instance `forall 0:X0=128 /\ [x]=127`.